client/main.lua

```diff
@@ -117,7 +142,7 @@ function toggleVoice(plySource, enabled, moduleType)
 				end
 			end)
 		end
-		MumbleSetVolumeOverrideByServerId(plySource, enabled and volume or -1.0)
+		MumbleSetVolumeOverrideByServerId(plySource, enabled and volumel or -1.0)
```

Comment:
this seems to be my left over code, this will always default to -1.0 because this will only happen when its set to false.